src/rootReducer.js

```diff
@@ -22,41 +23,45 @@ const buildBoard = (state, numberOfElements) => {
   }
   shuffle(board)
 
-  return { ...state, phase: PLAY, gameBoard: board }
+  return { ...state, phase: PLAY, duration: 0, gameBoard: board }
 }
 
 const markMatchedCards = (state, matchValue) => {
-  const newGameBoard = state.gameBoard.map(card => {
-    if(card.value === matchValue) {
-      card.matched = true
-      return card
-    } else {
-      return card
-    }
-  })
+  const newGameBoard = state.gameBoard.map(card => (
+    card.value === matchValue ? 
+    { ...card, matched: true } :
+    card
+  ))
   if(validateGameEnd(newGameBoard)) {
+      axios.post('https://salty-headland-84520.herokuapp.com/scores', 
```

Comment:
Tutaj tym bardziej nie robimy takich rzeczy. Reducer ma za zadanie przyjąć dane -> obrobuć ja na nasze potrzebny i zapisać do store, to koniec. W tym momencie nie mamy kontroli co się stanie z naszą apką :open_mouth: co w przypadku kiedy pójdzie jakiś błąd? :thinking: 

Musimy sobie ogarnać nową warstwę, gdzie będą zapytania do API